definitions/npm/reselect_v3.x.x/test_reselect_3.x.x.js

```diff
@@ -42,50 +39,69 @@ createSelector(
   (state: State, props: TestProps) => state.x + props.x,
   (state: State, props: TestProps) => state.y + props.x,
   (x, y) => {
-    return x + y
+    return x + y;
   }
-)({x: 100, y: 200}, {
-  x: 10
-});
+)(
+  { x: 100, y: 200 },
+  {
+    x: 10
+  }
+);
 // END TEST
 
 // TEST: Should pass for additional arguments
 createSelector(
   (state: State, props: TestProps, test) => state.x + props.x + test,
   (state: State, props: TestProps, test) => state.y + props.x + test,
   (x, y) => {
-    return x + y
+    return x + y;
   }
-)({x: 100, y: 200}, {
-  x: 10
-}, 10);
+)(
+  { x: 100, y: 200 },
+  {
+    x: 10
+  },
+  10
+);
+// END TEST
+
+// TEST: Should work when using another selector as functions
+const simpleSelector = (state: State): string => "foo";
+const resultSelector = (arg: string): string => "foo";
+
+const combinedSelector1 = createSelector(simpleSelector, resultSelector);
+
+const combinedSelector2: (state: State) => string = createSelector(
+  combinedSelector1,
+  resultSelector
+);
```

Comment:
this is the only new test. This comes from https://github.com/facebook/flow/issues/4959